draft-ietf-gnap-core-protocol.md

```diff
@@ -3261,12 +3213,99 @@ the token management URL as valid, since the end result is still the token
 not being usable.
 
 
-# Using Access Tokens {#use-access-token}
+# Securing Requests from the Client Instance {#secure-requests}
+
+In GNAP, the client instance secures its requests to the AS and RS by presenting an access
+token, presenting proof of a key that is possesses, or both an access token and
+key proof together.
+
+- When an access token is used with a key proof, this is a bound token request. This type of
+    request is used for calls to the RS as well as the AS during negotiation.
+- When a key proof is used with no access token, this is a non-authorized signed request. This
+    type of request is used for calls to the AS to initiate a negotiation.
+- When an access token is used with no key proof, this is a bearer token request. This type of
+    request is used only for calls to the RS.
+- When neither an access token nor key proof are used, this is an unsecured request. This 
+    type of used only for calls the RS during a discovery phase as
+    described in {{rs-request-without-token}}.
+
+## Key Formats {#key-format}
+
+Several different places in GNAP require the presentation of key material 
+by value. Proof of this key material MUST be bound to a request, the nature of which varies with
+the location in the protocol the key is used. For a key used as part of a client instance's initial request
+in {{request-client}}, the key value is the client instance's public key, and
+proof of that key MUST be presented in that request. For a key used as part of an
+access token response in {{response-token-single}}, the proof of that key MUST
+be used when presenting the access token.
+
+A key presented by value MUST be a public key in at least one
+supported format. If the key is sent in multiple
+formats, all the keys MUST be the same. 
+
+proof (string)
+: The form of proof that the client instance will use when
+    presenting the key. The valid values of this field and
+    the processing requirements for each are detailed in 
+    {{binding-keys}}. This field is REQUIRED.
 
-The method the client instance uses to send an access token to the RS depends on the value of the
-"key" and "proof" parameters in [the access token response](#response-token-single).
+jwk (object)
+: Value of the public key as a JSON Web Key. MUST
+            contain an "alg" field which is used to validate the signature.
+            MUST contain the "kid" field to identify the key in the signed
+            object.
+
+cert (string)
+: PEM serialized value of the certificate used to
+            sign the request, with optional internal whitespace.
 
-If the key value is the boolean `false`, the access token is a bearer token
+cert#S256 (string)
```

Comment:
This is not a key type (it's not a key!) so should not be listed here. Or if it is, we need to explain why.

Reply:
It's enough information for the AS/RS to validate the key presented in the TLS layer matches the one referenced here. We can expand the language on what goes in this object if that's helpful.

Reply:
Yes please. Something like: This "key type" is a reference to the key presented at the TLS layer.

Reply:
I've tried to clarify the nature of this value, and also removed the words "key type" that I think were misleading.